content/kubeone/master/tutorials/creating_clusters_baremetal/_index.en.md

```diff
@@ -3,7 +3,327 @@ title = "Creating a Kubernetes Cluster on Bare-metal"
 date = 2021-02-10T12:00:00+02:00
 weight = 2
 enableToc = true
-draft = true
 +++
 
-TBD
+In this tutorial, we're going to show how to use Kubermatic KubeOne to create
```

Comment:
Can you add this tutorial to the v1.3 docs as well? The placeholder file should already exist, and the content should be the same.